src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -643,74 +643,104 @@ private static object SplitOperatorImpl(ExecutionContext context, IScriptExtent
             }
         }
 
+        private static string ReverseStringUtility(string str)
```

Comment:
Can you add comments about what the functions do?  There is a good example on line 429

Reply:
I'm working on a fix right now that eliminates the ReverseStringUtility entirely. It should be pushed in the next few days, hopefully. Thank you, thanks for the tip though.